apps/builder/app/shared/instance-utils.test.tsx

```diff
@@ -3165,6 +3171,101 @@ describe("insert webstudio fragment copy", () => {
       availableDataSources: new Set(),
     });
     expect(data.instances.size).toEqual(4);
-    expect(newInstanceIds.size).toEqual(4);
+    expect(newInstanceIds.size).toEqual(5);
+  });
+});
+
+describe("support copy paste of :root styles", () => {
+  const css = (strings: TemplateStringsArray, ...values: string[]) => {
+    let css = "";
```

Comment:
having function called css and this variable, is not very nice for the reader

Reply:
this is temporary to make actual test more readable